src/main/java/org/opensearch/ad/settings/AnomalyDetectorSettings.java

```diff
@@ -793,8 +793,8 @@ private AnomalyDetectorSettings() {}
     // ======================================
     // Validate Detector API setting
     // ======================================
-    public static final long TOP_VALIDATE_TIMEOUT_IN_MILLIS = 60_000;
-    public static final long MAX_INTERVAL_REC_LENGTH_IN_MINUTES = 120L;
+    public static final long TOP_VALIDATE_TIMEOUT_IN_MILLIS = 10_000;
```

Comment:
Why decrease so much? Most of the API-level timeouts are 60s. Given a cluster under heavy load with a lot of data, couldn't this end up timing out a lot? 

I'm not sure on the details of your performance testing, but given all of the potential queries you're making, I can imagine this can easily take 2-3s given just a normal amount of data.